app/src/main/java/com/framgia/soundcloud/screen/listtracks/ListtracksViewModel.java

```diff
@@ -42,6 +43,7 @@ public void onStop() {
     public void onGetTrackSuccess(List<Track> tracks) {
         setLoading(false);
         setAdapter(new ListtrackAdapter(tracks, this));
+        this.tracks = tracks;
```

Comment:
ERROR: [STYLE][ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD] Write to static field com.framgia.soundcloud.screen.listtracks.ListtracksViewModel.tracks from instance method com.framgia.soundcloud.screen.listtracks.ListtracksViewModel.onGetTrackSuccess(List)
http://findbugs.sourceforge.net/bugDescriptions.html#ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD